Jellyfin.Api/Models/UserDtos/TxtLyricsProvider.cs

```diff
@@ -0,0 +1,81 @@
+using System;
+using System.Collections.Generic;
+using System.Collections.ObjectModel;
+using System.Dynamic;
+using System.Globalization;
+using System.Linq;
+using LrcParser.Model;
+using LrcParser.Parser;
+using MediaBrowser.Controller.Entities;
+
+namespace Jellyfin.Api.Models.UserDtos
+{
+    /// <summary>
+    /// TXT File Lyric Provider.
+    /// </summary>
+    public class TxtLyricsProvider : ILyricsProvider
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="TxtLyricsProvider"/> class.
+        /// </summary>
+        public TxtLyricsProvider()
+        {
+            FileExtensions = new Collection<string>
+            {
+                "lrc", "txt"
+            };
+        }
+
+        /// <summary>
+        /// Gets a value indicating the File Extenstions this provider works with.
+        /// </summary>
+        public Collection<string>? FileExtensions { get; }
+
+        /// <summary>
+        /// Gets or Sets a value indicating whether Process() generated data.
+        /// </summary>
+        /// <returns><c>true</c> if data generated; otherwise, <c>false</c>.</returns>
+        public bool HasData { get; set; }
+
+        /// <summary>
+        /// Gets or Sets Data object generated by Process() method.
+        /// </summary>
+        /// <returns><c>Object</c> with data if no error occured; otherwise, <c>null</c>.</returns>
+        public object? Data { get; set; }
+
+        /// <summary>
+        /// Opens lyric file for [the specified item], and processes it for API return.
+        /// </summary>
+        /// <param name="item">The item to to process.</param>
+        public void Process(BaseItem item)
+        {
+            string? lyricFilePath = Helpers.ItemHelper.GetLyricFilePath(item.Path);
+
+            if (string.IsNullOrEmpty(lyricFilePath))
+            {
+                return;
+            }
+
+            List<Lyric> lyricsList = new List<Lyric>();
+
+            string lyricData = System.IO.File.ReadAllText(lyricFilePath);
```

Comment:
Once you hook it up to the DI pipeline plugins will be possible so this looks great to me. My last comment is that there seems to be a mix of `lyric` and `lyrics` for variables and other references - we should pick one now and stick to it.

Reply:
Since `ISubtitleProvider` is already singular we should probably do that here, but as long as it's consistent everywhere I could go either way.

Reply:
I've gone through all the references and updated filename, variables, references, documentation, etc.

You have lyric providers. Lyric files. A single line from a song is a Lyric.

Songs have Lyrics. You get Lyrics for a song.